docs/Release Notes/release-notes.md

```diff
@@ -47,7 +128,7 @@ Find more details in [paralus/dashboard](https://github.com/paralus/dashboard).
 
 ---
 
-## v0.1.3
+## v0.1.4
 
 **August 26,2022**
```

Comment:
There is no release on August 26 as per https://github.com/paralus/helm-charts/releases
v0.1.4 is released on September 20.

Reply:
There was a v0.1.3 release which happened on Aug 26th for paralus and Aug 29th for helm charts. Since we are maintaining helm version here.. let's have an entry for v0.1.3 with Aug 29th as release date with below contents.

https://github.com/paralus/paralus/releases/tag/v0.1.3

https://github.com/paralus/relay/releases/tag/v0.1.1

https://github.com/paralus/prompt/releases/tag/v0.1.1

Reply:
v0.1.4 was helm only change.. with below change

Cleanup pod for kratos restart on success
https://github.com/paralus/helm-charts/pull/58